### ISSUE TITLE
Add Read Aloud AT to Understanding WCAG for SC 3.1.6

### ISSUE THREAD

**Issue by JaninaSajka** [w3c/wcag#1747]
It's not just screen reader users who are adversely affected by inadequate pronunciation. Users of read aloud AT (such as TextHelp) are also adversely affected and should be acknowledged. This is a simple way to acknowledge one segment of COGA issues.

**Comment by JaninaSajka** [comment]
My bad. UW 2.1 does seem to acknowledge read aloud tools, if not by name. Sorry, seems I was looking at an older Understanding at first.
